app/code/community/MVentory/Productivity/Helper/Data.php

```diff
@@ -222,4 +222,45 @@ public function sendEmail ($subject,
   public function isMVentoryModuleEnabled () {
     return parent::isModuleEnabled('MVentory_API');
   }
+
+  /**
+   * Get Images html
+   *
+   * @param int $productId
+   * @return string $html
+   */
+  public function getProductImagesToHtml($productId = null){
+    $html = '';
+    if ($productId != null) {
+      $_product = Mage::getSingleton('catalog/product')->load($productId);
+      $base_url = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_MEDIA);
+      foreach ($this->_getProductMediaGallery($productId) as $imageUrl) {
+        $u_id = uniqid();        
+        $html .='<li id="image_'.$u_id.'" class="qq-upload-success product-media-image-gallery"  
+              image="'.$imageUrl.'" 
+              style="background-image: url('.$base_url.'catalog/product'.$imageUrl.')"></li>';
```

Comment:
What if images are served from CDN, for example?
Please, use standard Magento functionality to generate URLs for images
Also, there's no need to load full-sized images here